CHANGELOG.md

```diff
@@ -403,7 +403,7 @@ Header Changes:
   * If you used files as views with `seqan3::views::get<seqan3::field::id>` to project a single field, e.g.
     * `seqan3::views::get<seqan3::field::id>(fin)` => `std::views::transform(fin, [](auto && record){ return record.id(); })`
     * `fin | seqan3::views::get<seqan3::field::id>()` => `fin | std::views::transform([](auto && record){ return record.id(); })`
-    * or per projection: `fin | std::views::transform(&decltype(fin)::record_type::id)`
+    * or per projection: `fin | std::views::transform(&decltype(fin)::%record_type::id)`
```

Comment:
was this intentional?

Reply:
Escaping some doxygen auto-link generation: https://doxygen.nl/manual/autolink.html#linkfunc
(Which apparently didn't work before)